day00/ex10/ex10.md

```diff
@@ -0,0 +1,57 @@
+# Exercise 10 - Vectorized Mean Squared Error as a linear cost function
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex10              |
+|   Files to turn in :    |  vec_linear_mse.py  |
+|   Forbidden function :  |  None              |
+|   Remarks :             |  n/a               |
+
+**Objective:**
+You must implement the following formula as a function:  
+
+$$
+MSE = \frac{1}{m}\sum_{i=1}^{m}(X\theta - y)^{T}}(X\theta - y)
+$$
+
+where:
+- $x$ is a matrix of dimension m * n,
+- $\theta$ is a vector of dimension n,
+- $y$ is a vector of dimension m,
+
+**Instructions:**
+In the linear_mse.py file create the following function as per the instructions given below:
+```python
+def vec_linear_mse(y, x, theta):
+    """Computes the mean squared error of three non-empty numpy.ndarray, without any for-loop. The three arrays must have compatible dimensions.
+    Args:
+      y: has to be an numpy.ndarray, a vector of dimension m * 1.
+      x: has to be an numpy.ndarray, a matrix of dimesion m * n.
+      theta: has to be an numpy.ndarray, a vector of dimension n * 1.
+    Returns:
+      The mean squared error as a float.
+      None if y, x, or theta are empty numpy.ndarray.
+      None if y, x or theta does not share compatibles dimensions.
+    Raises:
+      This function shouldn't raise any Exception.
```

Comment:
sap

Reply:
Not in this case : the dot product imply doing a transpose : a dot b = a^{T}b

No ?